### DISCUSSION TITLE
Sticky navbar

### DISCUSSION THREAD

**Discussion by LITUATUI** [luizdepra/hugo-coder#775] [Ideas]
Please add the option to make the navbar sticky, or make it default.

Thanks!

**Comment by luizdepra** [comment]
Hey!

This is kind of easy to do with a CSS rules, I think.
I don't want to bloat the theme with every possible customization, so I recommend you to try customizing it yourself. Try changing our SCSS files or create a new (S)CSS to override the default rules.
